### PR TITLE
flask support for python reviewer

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: packages/python-packages/apiview-gpt/app.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from flask import Flask, request, jsonify
+from src import review_python
+
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    return 'Hello, World!'
```
Comment (tjprescott) [inline packages/python-packages/apiview-gpt/app.py:8]: Is this really necessary?

File: packages/python-packages/apiview-gpt/src/_python_api.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from ._gpt_reviewer import GptReviewer
+
+def review_python(code):
```
Comment (tjprescott) [inline packages/python-packages/apiview-gpt/src/_python_api.py:3]: This should really accept code and language, as apiview-gpt should be able to work with any language. APIView (web) would be responsible for passing the appropriate language.

**Review by tjprescott — APPROVED** [review]
Some comments, but we can always tweak this code later.

File: packages/python-packages/apiview-gpt/app.py (hunk 1 above)
Comment (xiangyan99) [reply packages/python-packages/apiview-gpt/app.py:8]: I use it to detect if service is online.

I will remove it.

File: packages/python-packages/apiview-gpt/src/_python_api.py (hunk 2 above)
Comment (xiangyan99) [reply packages/python-packages/apiview-gpt/src/_python_api.py:3]: In the plan, this file will contain all Python related services. e.g. BreakingChangeDetector. And .net, js, java will have their own _language_api.py.

**Review by tjprescott — APPROVED** [review]

**Pull request merged**